src/main/scala/mimir/statistics/DetectSeries.scala

```diff
@@ -44,13 +44,47 @@ class DetectSeries(db: Database) {
         case _ => None
       }
     }
-//    Not working! Typechecker.schemaOf is not detecting type for csv loaded files( detected as 'Any')
-//    val seqCol = Typechecker.schemaOf(oper) 
     val seqCol = db.bestGuessSchema(oper)
     
-    val seriesColTemp = seqCol.flatMap( tup => getType(tup._1, tup._2))
+    val seriesColDate = seqCol.flatMap( tup => getType(tup._1, tup._2))
+
+    var series = seriesColDate.toList
+    
+    val seriesColNumIdx: Seq[Int] = seqCol.zipWithIndex.filter(tup => Seq(TInt(),TFloat()).contains(tup._1._2)).map(x => x._2).toSeq
     
-    seriesColTemp.toList
+    if(!seriesColNumIdx.isEmpty){
+      db.query(oper){result =>
+//        var prevVal = new ArrayBuffer[PrimitiveValue](seriesColNumIdx.size)
+        val initRow = result.next
+        var prevVal = seriesColNumIdx.map(initRow.tuple(_)).toArray
+        var countInc = new Array[Double](seriesColNumIdx.size)
+        var countDec = new Array[Double](seriesColNumIdx.size)
+        var totalTup = 1;
+        result.foreach(row => 
```

Comment:
While it varies based on situation, in general I've found the for loop syntax to be more readable:
```
for(row <- result){

}
```